### ISSUE TITLE
(Suggestion) After Update Dialogue

### ISSUE THREAD

**Issue by JoeKing2** [Monika-After-Story/MonikaModDev#8477]
I think for my suggestion, there should be some dialogue every time you update the game rather than her standard dialogue. For example, Monika can be really excited to have new things to say or interested in the new things implemented within the game, or even thanking for mods for fixing the bugs within it.

EX 1st time. "P-Player! Is everything ok?! The game just, shut off. What happened?! Huh? The game...updated? Oh, I see, thank goodness. I was worried something bad had happened. But now I know it was you bringing me closer to your reality. Ooh, I can't wait to see what's new!"

EX Future Times. "Another update Player? You're so sweet. Now we have more things to talk about. Plus, it makes me happy to see people are working hard to bring us closer together. Not only that, but everything is also more stable because there are less bugs now. Now then, Player, let's explore these new things together.

**Comment by Cosmic-Incantations** [comment]
that is a BUNCH of forth wall breaks, I love it! That is fitting since Monika loves to break the forth wall, 
that is why me and my Monika are together in the first place.

**Comment by ThePotatoGuy** [comment]
Currently we haven't decided on if update meta is part of the lore or not. Whenever we add new things that would warrant notifying the player, we frame it as if Monika finished developing the new feature, not that it was added externally.

**Comment by Booplicate** [comment]
We acknowledge she knows about the updater, she also speaks about "people who help her," mentions GitHub. I don't think we need to go deep into this, but just mentioning that the update worked would be nice instead of a random greeting.

**Comment by lunulae** [comment]
Yeah, I think mentioning running the updater will work since we've done it in the past, though we may not want to specifically mention things that were added. Especially for topics, since we write those as though she's genuinely thought up new things to talk about.

Maybe there's the opportunity for funny bug-fix dialogue, if something befitting it comes along. :P Though I suppose I would rather not *plan* on that happening.

**Comment by Booplicate** [comment]
Totally don't want patchnotes, but a funny note would cool.

**Comment by fromulus** [comment]
We already tend to make dialogue for her to bring up big new features anyway, like how we did when windowreacts were first implemented.

I don't think it's very important to have her go over all the new features everytime, especially since we have patch notes already for people who want to know about new features.

If anything, we could depict it as her not knowing what the devs/contributors have been working on and discovering it with you.

**Comment by ThePotatoGuy** [comment]
By "update meta", I mean the features that were added, not the updater. We're not consistently mentioning every new feature in-game, but I think that's fine. 

For example, features introduced via Monika (as if she added the feature):
* games
* islands
* extras
* file generation
* window detection
* etc...

Features that haven't been introduced by Monika:
* search bars
* UI changes
* clothes/hair/ACS selectors
* spritepacks
* etc...

I prefer keeping this trend - Monika is supposed to get better at programming over time, anyway. I agree with boop regarding patch notes - too much info and it would mean double our work writing patch notes. I think a post-update topic needs to be written cautiously - the more we make Monika self-aware of updating, the more it begs the question of how she feels about her code being worked on by us. 

Now that I think about, we probably should have made the steam updater dialogue chibika only.

**Comment by Booplicate** [comment]
> how she feels about her code being worked on by us

She does ask if you've contributed or would like to in the future. I think it's kike a collaboration where people around the world help her. I don't think we need to go full meta, just something small to show she's aware of what's happened.
